include/ttmlir/Dialect/TTIR/Analysis/LegalGridAnalysis.h

```diff
@@ -15,6 +15,7 @@ struct LegalGridAnalysisInput {
   ChipDescAttr chipDesc;
   GridAttr maxGrid;
   RankedTensorType tensorType;
+  int64_t maxShardedGrids = 64;
```

Comment:
constexpr

Reply:
I was intending for this to be a passable param. Will set to constexpr for now and change when needed.

Reply:
Indeed it should be passed in as param, make it as part of override issue in followup change.